ChaosMod/Effects/db/Player/PlayerTpController.cpp

```diff
@@ -386,4 +386,75 @@ REGISTER_EFFECT(OnStartFakeTp, nullptr, nullptr, EffectInfo
 		.Name = "Fake Teleport",
 		.Id = "tp_fake"
 	}
+);
+// clang-format on
+
+static void OnStartFakeFakeTp()
+{
+	FakeTeleportInfo selectedLocationInfo = tpLocations.at(g_Random.GetRandomInt(0, tpLocations.size() - 1));
+	auto overrideId                       = selectedLocationInfo.type;
+	GetComponent<EffectDispatcher>()->OverrideEffectNameId("tp_fakex2", overrideId);
+
+	Player player     = PLAYER_ID();
+	Ped playerPed     = PLAYER_PED_ID();
+	Vehicle playerVeh = IS_PED_IN_ANY_VEHICLE(playerPed, false) ? GET_VEHICLE_PED_IS_IN(playerPed, false) : 0;
+
+	Vector3 playerPos = GET_ENTITY_COORDS(playerPed, false);
+
+	Hooks::EnableScriptThreadBlock();
```

Comment:
Script thread block enables, but never disables

Reply:
Whoops! Sometimes I forget things, that should be fixed in the latest commit.